packages/frontend/src/redux/slices/multiplier/index.js

```diff
@@ -0,0 +1,64 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import { wallet } from "../../../utils/wallet";
+import { createSelector } from "reselect";
+import { ACCOUNT_ID_SUFFIX } from "../../../config"
+
+
+
+const SLICE_NAME = "multiplier";
+
+const initialState = {
+    prices: {},
```

Comment:
What do you think about having this slice have a similar shape to `tokenFiatValues` so that it is easily extendable?